### DISCUSSION TITLE
how to deploy many models by openai_api.py

### DISCUSSION THREAD

**Discussion by samosun** [QwenLM/Qwen#1221] [Q&A] [answered]
@jklj077 I succesfully deploy a qwen model by openai_api.py , how can I deploy more models in same api , so that I can choose model by model_name in client , eg: llm_1 = ChatOpenAI(model_name="Qwen-1.8B",openai_api_base='same_api'), llm_2 = ChatOpenAI(model_name="Qwen-14B",openai_api_base='same_api')

**Comment by jklj077** [comment, marked as answer]
I don't think it is supported. Frameworks like FastChat are more likely to support this usecase.
